fix: Can't make search by email in usersmanagement - EXO-65032 - Meeds-io/meeds#1085

akhanfir opened the pull request:
Prior to this change, when open usersManagement interface and search for users by their mail or login, search shows no results.To fix this problem, added the search by email or login in the search query.After this change, In user management search, should retrieve users by first&last names, emails & usernames.

akhanfir committed:
fix: Can't make search by email in usersmanagement - EXO-65032 - Meeds-io/meeds#1085
Prior to this change, when open usersManagement interface and search for users by their mail or login, search shows no results.To fix this problem, added the search by email or login in the search query.After this change, In user management search, should retrieve users by first&last names, emails & usernames.

ahamdi commented:
@akhanfir Could you resolve conflicts ?

rdenarie reviewed (changes requested):
Can you fix the sonar bug ?
In addition can you try to add a unit test to cover the added lines ?

Thanks

exo-swf commented:
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

sofyenne committed:
fix : Display the default user avatar for disabled users - EXO-66363 - Meeds-io/meeds#1129 (#2940)

This change is going to display the default user avatar for the disabled users.

akhanfir committed:
fix : Display the default user avatar for disabled users - EXO-66363 - Meeds-io/meeds#1129 #2940
This change is going to display the default user avatar for the disabled users.

ahamdi commented on component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java:
```diff
@@ -338,6 +339,7 @@ public Response getUsers(@Context UriInfo uriInfo,
       Identity target = identityManager.getOrCreateIdentity(OrganizationIdentityProvider.NAME, userId);
       ProfileFilter filter = new ProfileFilter();
       filter.setName(q == null || q.isEmpty() ? "" : q);
+      filter.setSearchEmail(searchEmail ? true : false);
```
```suggestion
      filter.setSearchEmail(searchEmail);
```

exo-swf commented:
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

exo-swf commented:
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

IlhemEssaadi committed:
Add the space administration portlet to the ManageSpaceGRP load group - EXO-68206 - Meeds-io/meeds#1467

Prior to this change, the site navigation icon is not displayed on the space administration page due to the hide of sharedLayout for the aggregated site containing the site navigation portlet.This commit creates a load-group for site navigation, which will then be loaded with the Spaces Administration Portlet.

Jihed525 commented:
@rdenarie can you have a look ?
FOr test coverage, @akhanfir confirms that it will be handled in the next pr https://github.com/Meeds-io/social/pull/2776

rdenarie reviewed (approved):
ok for me
